rubocop-rspec.gemspec

```
@@ -34,8 +34,10 @@ Gem::Specification.new do |spec|
   spec.metadata = {
     'changelog_uri' => 'https://github.com/rubocop/rubocop-rspec/blob/master/CHANGELOG.md',
     'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec/',
-    'rubygems_mfa_required' => 'true'
+    'rubygems_mfa_required' => 'true',
+    'default_lint_roller_plugin' => 'RuboCop::RSpec::Plugin'
   }
 
-  spec.add_dependency 'rubocop', '~> 1.61'
+  spec.add_dependency 'lint_roller', '~> 1.1'
+  spec.add_dependency 'rubocop', '~> 1.72.1'
```

I think we want to allow rubocop 1.73+ as well:
```suggestion
  spec.add_dependency 'rubocop', '~> 1.72', '>= 1.72.1'
```

---

Oops, I've fixed it 🙇